crates/ratchet-core/src/storage/cpu_buffer.rs

```diff
@@ -95,7 +95,7 @@ impl CPUBuffer {
     }
 
     pub fn to_slice<T: NoUninit + Pod>(&self, shape: &Shape) -> &[T] {
-        assert_eq!(self.n_bytes(), shape.numel() * std::mem::size_of::<T>());
+        // assert_eq!(self.n_bytes(), shape.numel() * std::mem::size_of::<T>());
```

Comment:
Can't comment out this load bearing check, if we need an unsafe version that's cool.

Reply:
Fixed